lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -81,6 +81,29 @@ static Type lowerType(Type type) {
   return {};
 }
 
+/// Return true if the specified FIRRTL type is a sized type (Int or Analog)
+/// with zero bits.
+static bool isZeroBitFIRRTLType(Type type) {
+  return type.cast<FIRRTLType>().getPassiveType().getBitWidthOrSentinel() == 0;
```

Comment:
Is the `getPassiveType` conversion necessary?

_Edit:_ I now see that this was just moved from lower in the file. Don't change this in this PR if it is wrong. 😉